src/voice_annotation_tool/application.py

```diff
@@ -14,7 +14,8 @@
 
 def get_data_dir() -> Path:
     """Returns the directory where application data is stored."""
-    data_dir = Path(QStandardPaths.standardLocations(QStandardPaths.AppDataLocation)[0])
+    data_dirs = QStandardPaths.standardLocations(QStandardPaths.AppDataLocation)
```

Comment:
Is this the same on linux and windows?

Reply:
`standardLocations` return the OS-specific folders. I have tested it on Linux and Windows, in Windows it is found in the AppData folder.

Reply:
as discussed, AppDataLocalLocation might be better.